src/drivers/imx/sai.c

```diff
@@ -250,15 +276,15 @@ static inline int sai_set_config(struct dai *dai, struct ipc_config_dai *common_
 		 */
 		val_cr2 |= REG_SAI_CR2_BCP;
 		val_cr4 |= REG_SAI_CR4_FSE;
-		val_cr4 |= REG_SAI_CR4_SYWD(0U);
+		val_cr4 |= REG_SAI_CR4_SYWD(1U);
 		break;
 	case SOF_DAI_FMT_DSP_B:
 		/*
 		 * Frame high, one bit for frame sync,
 		 * frame sync asserts with the first bit of the frame.
 		 */
 		val_cr2 |= REG_SAI_CR2_BCP;
-		val_cr4 |= REG_SAI_CR4_SYWD(0U);
+		val_cr4 |= REG_SAI_CR4_SYWD(1U);
 		break;
```

Comment:
Not sure I understand this change. Is this a bugfix? We should add it in a separate patch with proper explanation.

Reply:
I added a separate commit. Or did you mean put it into a separate pull request?